docs/usage/self-hosted-configuration.md

```diff
@@ -15,6 +15,22 @@ Please also see [Self-Hosted Experimental Options](./self-hosted-experimental.md
 
 ## allowCustomCrateRegistries
 
+## allowExtraConfig
+
+This option is only available via command line option `--allow-extra-config`.
```

Comment:
This seems to indicate that `cli` should be set to `true` in lib/config/options/index.ts. Then this line can be deleted.

Same goes for the other property.